### PR TITLE
Removes broken fake data generation

### PR DESCRIPTION
**Pull request by willfish**
### Jira link

P4-<TODO>

### What?

I have added/removed/altered:

- [x] Removes broken cases for journey fake data that don't map onto real ways of redirecting journeys (an impossible event - we only redirect moves)

### Why?

I am doing this because:

- This is an impossible state to get into

### CODE REVIEW

File: lib/tasks/fake_data/journeys.rb [hunk 1]
```diff
@@ -177,12 +118,8 @@ def create_redirect_move_event(timestamp, notes, location)
       end
 
       def random_event
-        case rand(1..6)
+        case rand(1..2)
```
Comment (martyn-w) [inline lib/tasks/fake_data/journeys.rb:121]: how about `if rand(0..1) == 1`

**Review by martyn-w — APPROVED** [review]
👍

File: lib/tasks/fake_data/journeys.rb (hunk 1 above)
Comment (martyn-w) [reply lib/tasks/fake_data/journeys.rb:121]: actually -just delete this `random_event()` method and move the if up into the main method ^

Comment (willfish) [reply lib/tasks/fake_data/journeys.rb:121]: We mock this method in the test so requires updating. Leaving as is painful/not my main priority atm. Will likely remove this/refactor later.

**Pull request merged**